<launch>
    <group>
    <node pkg="dobot" type="DobotServer" name="DobotServer" required="true" args="port_dobot" output="screen"/>
    <node pkg="dobot" type="DobotUse.py" name="dobot_service" output="screen"/>
    </group>
</launch>